src/connections/allContacts.js

```diff
@@ -104,3 +102,18 @@ export const withContactsMutations = withMutations(client => ({
 }))
 
 export default flow([withContacts, withContactsMutations])
+
+export function mergeContact(contact, attributes) {
+  function concat(arr1 = [], arr2 = []) {
+    return [...arr1, ...arr2]
+  }
+  function customizer(objValue, srcValue, key) {
+    if (Array.isArray(objValue) || Array.isArray(srcValue)) {
+      return concat(objValue, srcValue)
+    }
+    if (key === 'fullname') {
```

Comment:
Couldn't this be generalized to other string attributes? (I'm not sure, just asking)

Also what happens to them with the current implementation?

Reply:
Actually I don't remember of other top-level string attributes... Are there any?